Neos.Flow/Tests/Functional/Security/Authorization/Privilege/Entity/Doctrine/ContentSecurityTest.php

```diff
@@ -235,7 +243,7 @@ public function customersCannotSeeOthersRestrictableEntites()
         $otherAccount = new Security\Account();
         $otherAccount->setAccountIdentifier('othersAccount');
         $otherAccount->setAuthenticationProviderName('SomeProvider');
-        $this->persistenceManager->add($ownAccount);
+        $this->persistenceManager->update($ownAccount);
```

Comment:
Why did this change?
Since `$ownAccount` is a new instance `$this->persistenceManager->add($ownAccount)` makes more sense and it this needs to be adjusted it's probably due to some breaking change in behavior

Reply:
Because it's alreadyt added in the call for `authenticateRoles` - try and change it back to `add` and you will see the exception of "this is not a new object - use update instead"